mlflow/tracking/client.py

```diff
@@ -91,6 +91,12 @@ def _get_registry_client(self):
                 )
         return registry_client
 
+    def get_store(self):
```

Comment:
I don't think this needs to be a public API defined on `MlflowClient`. It looks like we're using this in `fluent.py` to fetch the track store, which presumably just involves mapping the MLflow Tracking URI to a store implementation. 

In that case, I think we can use `mlflow.tracking._tracking_service.utils._get_store` instead (see https://github.com/mlflow/mlflow/blob/8e6f8e363441d7f3e0d26b5127a29611d40aa9b4/mlflow/tracking/_tracking_service/utils.py#L126).

Reply:
Done!